applications/blockpublisher.md

```diff
@@ -0,0 +1,151 @@
+
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Overview
+#### Decentralized Publishing of News, Journals, and Books
+
+The advent of blockchain technology has the potential to transform the publishing industry in several ways. By leveraging decentralized networks and cryptographic techniques, blockchain-based publishing solutions can offer a more efficient, transparent, and accessible system for authors, publishers, and readers alike. It's also versatile enough to be used for entertainment content.
+
+
+
+### Project Details
+
+Key Features of Blockchain-Based Publishing:
+
+- Decentralized Price Discovery: Blockchain-based platforms can facilitate decentralized price discovery, allowing readers to directly support authors by contributing to a community-driven funding pool. This approach ensures that authors receive fair compensation for their work, while readers have greater control over the content they consume.
+- Subsidized Articles through Taxes: By implementing a token-based system, blockchain-based publishing platforms can introduce a novel approach to subsidizing articles. A small portion of the token's value can be allocated to support authors, making high-quality content more accessible to readers.
+- Free Staking Coin for Enhanced Accessibility: To further promote open access and sustainability, blockchain-based publishing platforms can introduce a free staking coin. This coin can be used to purchase articles, remove paywalls, or access premium content, making it more convenient for readers to engage with published works.
+- Open Access and Paywall Removal: The free staking coin can also be used to make articles open access or remove paywalls entirely. This approach can increase readership, promote knowledge sharing, and support the dissemination of high-quality research and content.
+
+## Make the content Open Access Model
+
+### Pasword Sharing:
+Sharing passwords for encrypted files is problematic, in P2P systems. One approach is to store the hash of the password in the blockchain and use Zero Knowledge Proof.
+
+### Secure Article Purchase:
+When a buyer purchases an article, they will pay double or 1.5 times the price. The money will remain in escrow, a secure and tamper-proof storage solution, until the buyer has proven they have access to the article.
+
+### Zero-Knowledge Proof:
+
+To facilitate this, a zero-knowledge proof system will be used. In this system, the buyer must prove to the network that they possess the correct password, without revealing the password itself.
+
+### Per-Article Fee and Open Access:
+
+A per-article fee will be chosen from a list of tips provided. The money fund for the article will remain in escrow until the price discovery process is complete. If the article receives additional money from taxes, it will be released from escrow and article is made open access to the public.
```

Comment:
> If the article receives additional money from taxes

I don't understand what this means. Taxes from who? Why do the taxes allow the article to be publicly released? Also, are people paying to read the article themselves, or are they paying to release the article to the public?

Reply:
Taxes I mean token collected to treasury through transaction and inflation. If article is for public good, they directly apply paying from treasury instead of user pay model.

> Also, are people paying to read the article themselves, or are they paying to release the article to the public?

Its both, they are paying to read article themself and also release the article to the public.

Reply:
thanks that makes more sense!